examples/glamorous-jest-react/index.test.js

```diff
@@ -0,0 +1,22 @@
+import {matcher, serializer} from 'jest-glamor-react'
+import React from 'react'
+import * as enzyme from 'enzyme'
+import toJson from 'enzyme-to-json'
+import {Wrapper, Title} from './index'
+
+expect.addSnapshotSerializer(serializer)
```

Comment:
I think this comment is a good solution. If I put it in a setup file for this example, it's significance might not be easily noticeable?